install.py

```diff
@@ -0,0 +1,10 @@
+import launch
+import os
+
+req_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "requirements.txt")
```

Comment:
Would you mind change this to use pathlib instead of os.path? Just to be consistent with the rest of the code

Reply:
Yeah no problem.